java/src/main/java/com/cybersource/ws/client/SecurityUtil.java

```diff
@@ -38,7 +36,7 @@ public class SecurityUtil {
     private static MessageHandlerKeyStore localKeyStoreHandler = null;
     
     //mapping between IdentityName and Identity
-    private static ConcurrentHashMap<String, Identity> identities = new ConcurrentHashMap<String, Identity>();
+    private static ConcurrentHashMap<String, Identity> identities = new ConcurrentHashMap<>();
```

Comment:
put this explicit type argument <String, Identity> back.  we still support java 6. And in java6 it is required.